net/frr/src/opnsense/service/templates/OPNsense/Quagga/bgpd.conf

```diff
@@ -95,6 +152,10 @@ router bgp {{ OPNsense.quagga.bgp.asnumber }}
 {%         if 'attributeunchanged' in neighbor and neighbor.attributeunchanged != '' %}
  neighbor {{ neighbor.address }} attribute-unchanged {{ neighbor.attributeunchanged }}
 {%         endif %}
+{%         if neighbor.peergroup is defined %}
```

Comment:
might be safer to check for ! ''

e.g. `if neighbor.peergroup|default('') != ''`

I think empty is also `defined`